client/src/components/Reviews/ReviewSummary/index.jsx

```diff
@@ -3,22 +3,13 @@ import PropTypes from 'prop-types';
 import AverageRating from './AverageRating.jsx';
 import RatingBreakdown from './RatingBreakdown.jsx';
 import ProductBreakdown from './ProductBreakdown.jsx';
+import sharedFns from '../../SharedComponents/sharedFns.js';
```

Comment:
I don't see the file sharedFns.js included in this PR.
Did you mean to change the path to the './client/src/utilityFns.js'?

Reply:
I have updated the file path and function name.